luarules/gadgets/include/DirectionsUtil.lua

```diff
@@ -19,86 +19,93 @@ DirectionsUtil.Directions = {}
-    { 1.266109423779690200e-01,  9.635507195418956400e-01,  -2.356685811483104400e-01, -5.918057918773704800e-01, 7.708607037331326500e-01,  2.356685811483104100e-01,  8.112044025054730700e-02,  6.173531063344418500e-01,  7.824925662731709900e-01,  -3.791737544816633500e-01, 4.938953812696635900e-01,  -7.824925662731708800e-01, -9.635507195418956400e-01, 1.266109423779690200e-01,  -2.356685811483104400e-01, -7.708607037331326500e-01, -5.918057918773704800e-01, 2.356685811483104100e-01,  -6.173531063344418500e-01, 8.112044025054730700e-02,  7.824925662731709900e-01,  -4.938953812696635900e-01, -3.791737544816633500e-01, -7.824925662731708800e-01, -1.266109423779690200e-01, -9.635507195418956400e-01, -2.356685811483104400e-01, 5.918057918773704800e-01,  -7.708607037331326500e-01, 2.356685811483104100e-01,  -8.112044025054730700e-02, -6.173531063344418500e-01, 7.824925662731709900e-01,  3.791737544816633500e-01,  -4.938953812696635900e-01, -7.824925662731708800e-01, 9.635507195418956400e-01,  -1.266109423779690200e-01, -2.356685811483104400e-01, 7.708607037331326500e-01,  5.918057918773704800e-01,  2.356685811483104100e-01,  6.173531063344418500e-01, -8.112044025054730700e-02, 7.824925662731709900e-01, 4.938953812696635900e-01, 3.791737544816633500e-01, -7.824925662731708800e-01 }
+	{},
+	{ 1,                         0,                         0,                         -1,                        0,                         0 },
+	{ 1,                         0,                         0,                         -0.5,                      0,                         0.866025403784439,         -0.5,                      0,                         -0.866025403784438 },
+	{ -0.577350269072,           0.577350269072,            -0.577350269072,           0.577350269072,            0.577350269072,            0.577350269072,            -0.577350269072,           -0.577350269072,           0.577350269072,            0.577350269072,            -0.577350269072,           -0.577350269072 },
+	{ -1.478255937088018300e-01, 8.557801392177640800e-01,  4.957700547280610200e-01,  9.298520676823500700e-01,  -3.330452755499895800e-01, -1.563840677968503200e-01, -7.820264758448114400e-01, -5.227348665222011400e-01, -3.393859902820995400e-01, -3.612306945786420600e-02, -5.056147808319168000e-01, 8.620027942282061400e-01,  3.612306958303366400e-02,  5.056147801034870400e-01,  -8.620027946502272200e-01 },
+	{ 0.2125482559,              -0.819507424,              0.5321943438,              -0.9771505706,             -0.1782579665,             0.115762077,               -0.2125482559,             0.819507424,               -0.5321943438,             0.9771505706,              0.1782579665,              -0.115762077,              0,                         0.544639035,               0.8386705679,              0,                         -0.544639035,              -0.8386705679 },
```

Comment:
Actually nevermind, the numbers are different. What was wrong with the previous value?

Reply:
They weren't random enough, I assume